### DISCUSSION TITLE
QUERY about .expression

### DISCUSSION THREAD

**Discussion by Tahmidanam** [giswqs/qgis-earthengine-examples#16] [Q&A] [answered]
Is it possible to apply following code for LST from LANDSAT in GEE_QGIS?

LST = thermal.expression(
    '(Tb/(1 + (0.001145* (Tb / 1.438))*log(Ep)))-273.15', {
      'Tb': thermal.select('B6'),
      'Ep': EM.select('EMM')
}).rename('LST')

**Comment by giswqs** [comment, marked as answer]
See this [example](https://github.com/giswqs/qgis-earthengine-examples/blob/master/Image/expressions.py).
